mozdownload/treeherder.py

```diff
@@ -21,48 +24,77 @@
     'win64': {'build_os': 'win', 'build_architecture': 'x86_64'},
 }
```

Comment:
Yeah, maybe we can do a `config.py` or such in the future, so we have a central place for all configuration settings.